.github/workflows/ci.yml

```diff
@@ -194,7 +194,63 @@ jobs:
           nix-shell: ci-linter
           gh_token: ${{ secrets.GITHUB_TOKEN }}
           cross-prefix: "aarch64-unknown-linux-gnu-"
-
+  config_variations:
+    name: Non-standard configurations
```

Comment:
Please add a comment here explaining what's happening in this job?

Reply:
Done

Reply:
Ok